api/proto/teleport/legacy/types/types.proto

```diff
@@ -7790,6 +7790,8 @@ message OktaOptions {
 message AccessGraphSync {
   // AWS is a configuration for AWS Access Graph service poll service.
   repeated AccessGraphAWSSync AWS = 1 [(gogoproto.jsontag) = "aws,omitempty"];
+  // Azure is a configuration for the Azure Access Graph service poll service.
+  repeated AccessGraphAzureSync Azure = 2 [(gogoproto.jsontag) = "azure,omitempty"];
```

Comment:
you can keep these out for now.

Reply:
Removed it on this assumption. Will add back once code is ready.